testing/testing.gradle

```diff
@@ -62,3 +62,5 @@ tasks.register("testDisabledFieldInjection", Test) {
   include "context/FieldBackedProviderFieldInjectionDisabledTest.class"
 }
 test.dependsOn(testDisabledFieldInjection)
+
+test.forkEvery = 1
```

Comment:
Honestly, I am not sure about that. I am trying to understand why our builds are so long and this constant forking is one of my top suspects right now. 

I would like to raise the problem of flaky and slow tests on the next SIG meeting.

Reply:
Good point, I just ran this with and without `forkEvery = 1`, and it's 50% slower with `forkEvery = 1` ☹️.

But I'm currently finding the flakiness more painful than the duration, so I propose we take this in as a temporary measure.

Reply:
Let us just hope that cure will not be worse than the sickness. And when we start performance optimisations, the first step that we will do, is to remove this fork settings :)